src/Console/Exporters/FluentExporter.php

```diff
@@ -51,6 +55,29 @@ public function exportClassMetadata(ClassMetadataInfo $metadata)
         return implode("\n", $lines);
     }
 
+    public function export()
+    {
+        if (!is_dir($this->_outputDir)) {
+            mkdir($this->_outputDir, 0775, true);
+        }
+
+        foreach ($this->_metadata as $metadata) {
+            // In case output is returned, write it to a file, skip otherwise
+            if ($output = $this->exportClassMetadata($metadata)) {
+                $path = $this->_outputDir.'/'.str_replace('\\', '.', $this->_mappingClassName).$this->_extension;
```

Comment:
is this the convention for fluent mappers? like `Namespace.Class` for `Namespace\Class` mappings?